src/schema/rules/top_level_files.yaml

```diff
@@ -3,6 +3,9 @@ README:
   required: true
   extensions:
   - None
+  - .md
+  - .rst
+  - .txt
```

Comment:
FWIW -- inspired me to file https://github.com/bids-standard/bids-specification/issues/1047